app/kumactl/cmd/manage/ca/provided.go

```diff
@@ -0,0 +1,176 @@
+		Long:  `Delete certificate authority.`,
+		RunE: func(cmd *cobra.Command, _ []string) error {
+			client, err := pctx.CurrentProvidedCaClient()
+			if err != nil {
+				return err
+			}
+			if err := client.DeleteCa(pctx.CurrentMesh()); err != nil {
+				return errors.Wrap(err, "could not delete certificate authority")
+			}
+			return nil
+		},
+	}
+	return cmd
+}
+
+func newCertificatesCmd(pctx *kumactl_cmd.RootContext) *cobra.Command {
+	cmd := &cobra.Command{
+		Use:   "certificates",
+		Short: "Manage certificates",
+		Long:  `Manage certificate.`,
+	}
+	// sub-commands
+	cmd.AddCommand(newAddCertificateCmd(pctx))
+	cmd.AddCommand(newListCertificatesCmd(pctx))
+	cmd.AddCommand(newDeleteCertificateCmd(pctx))
+	return cmd
+}
+
+type addCertificateContext struct {
+	*kumactl_cmd.RootContext
+
+	args struct {
+		keyFile  string
+		certFile string
+	}
+}
+
+func newAddCertificateCmd(pctx *kumactl_cmd.RootContext) *cobra.Command {
+	ctx := addCertificateContext{RootContext: pctx}
+	cmd := &cobra.Command{
+		Use:   "add",
+		Short: "Add signing certificate",
+		Long:  `Add signing certificate.`,
+		RunE: func(cmd *cobra.Command, _ []string) error {
+			client, err := ctx.CurrentProvidedCaClient()
+			if err != nil {
+				return err
+			}
+			certBytes, err := ioutil.ReadFile(ctx.args.certFile)
+			if err != nil {
+				return errors.Wrap(err, "could not read content of the certificate file")
+			}
+			keyBytes, err := ioutil.ReadFile(ctx.args.keyFile)
+			if err != nil {
+				return errors.Wrap(err, "could not read content of the key file")
+			}
+			pair := tls.KeyPair{
+				CertPEM: certBytes,
+				KeyPEM:  keyBytes,
+			}
+			signingCert, err := client.AddSigningCertificate(ctx.CurrentMesh(), pair)
+			if err != nil {
+				return err
+			}
+			cmd.Printf("Signing certificate add. Id: %s", signingCert.Id)
+			return nil
+		},
+	}
+	cmd.Flags().StringVar(&ctx.args.keyFile, "key-file", "", "path to a file with a private key")
+	_ = cmd.MarkFlagRequired("key-file")
+	cmd.Flags().StringVar(&ctx.args.certFile, "cert-file", "", "path to a file with a TLS certificate")
+	_ = cmd.MarkFlagRequired("cert-file")
+	return cmd
+}
+
+func newListCertificatesCmd(pctx *kumactl_cmd.RootContext) *cobra.Command {
+	cmd := &cobra.Command{
+		Use:   "list",
+		Short: "List signing certificates",
+		Long:  `List signing certificates.`,
+		RunE: func(cmd *cobra.Command, _ []string) error {
+			client, err := pctx.CurrentProvidedCaClient()
+			if err != nil {
+				return err
+			}
+			certs, err := client.SigningCertificates(pctx.CurrentMesh())
+			if err != nil {
+				return errors.Wrap(err, "could not retrieve signing certificates")
+			}
+			if err := printListCertificates(certs, cmd.OutOrStdout()); err != nil {
+				return errors.Wrap(err, "could not print certificates")
+			}
+			return nil
+		},
+	}
+	return cmd
+}
+
+func printListCertificates(certs []types.SigningCert, out io.Writer) error {
+	data := printers.Table{
+		Headers: []string{"ID", "CERT"},
```

Comment:
It's not helpful for a user to see raw CA cert bytes in the `CERT` column.

A useful info could include:
- Common Name
- Serial Number
- Not valid before
- Not valid after
- SHA-1 fingerprint
- SHA-256 fingerprint